nodewatcher/modules/frontend/ip_space/static/js/draw_cidrs.js

```diff
@@ -0,0 +1,265 @@
+window.DrawCidr = class DrawCidr {
+    constructor(svg, size, subnet, url) {
+        this.svg = svg;
+        this.loadedPrefixes = 0;
+
+        this.n_ips = Math.pow(2, 32 - parseInt(subnet.split("/")[1]));
```

Comment:
`'` instead `"`. Please when I make a comment, make sure yourself to check everywhere else for the same issue.